conf/messages

```diff
@@ -2976,60 +3000,82 @@ previousYearsLosses.capacitor.uk.helpText=You can include previous losses in thi
 previousYearsLosses.capacitor.nonUk.helpText=You can include previous losses in this return to reduce the person’s gain. You can only include other losses that have already been reported to HMRC. For example, losses on shares or other UK property sales or disposals.
 previousYearsLosses.personalRep.uk.helpText=You can include previous losses in this return to reduce the person’s gain. You can only include other losses that have already been reported to HMRC. For example, losses on shares or other UK property sales or disposals.
 previousYearsLosses.personalRep.nonUk.helpText=You can include previous losses in this return to reduce the person’s gain. You can only include other losses that have already been reported to HMRC. For example, losses on shares or other UK property sales or disposals.
+previousYearsLosses.personalRepInPeriodOfAdmin.uk.helpText=You can include previous losses in this return to reduce the estate’s gain.<br/>	You can only include the estate's other losses if they have already been reported to HMRC. For example, losses on shares or other UK property.
```

Comment:
these look like straight quotes in `estate's`

Reply:
fixed